index.html

```diff
@@ -20,79 +20,80 @@
       <nav id="navbar">
         <ul class="nav-links">
           <li><a href="#welcome-section" class="nav-link">Welcome</a></li>
-          <li><a href="#projects" class="nav-link">Projects</a></li>
+          <li><a href="#projects" class="nav-link">Projects</a>
```

Comment:
ERROR: Missing closing </li> tag.

Reply:
> ERROR: Missing closing tag.

:red_circle: You have well asked for this change 👌. But I think this comment is not clear enough to the reader to having missed a closing tag

You  could say: :arrow_down: 
- [ ]  Great job so far! 👍🏿 However, I've noticed that there is a missing closing tag. I kindly recommend adding the closing tag. Including a closing tag is important for proper HTML structure and to ensure that the elements are correctly nested and formatted. Without a closing tag, the HTML code may not render as intended or may result in errors. By adding the closing tag, you ensure that the HTML document is valid and follows the correct syntax

- Briefly for every section that needs an update try to point it out and tell the student what changes you expect from the section and why the change is needed.